chain/beacon/chain.go

```diff
@@ -158,7 +161,11 @@ func (c *chainStore) runChainLoop() {
 			c.l.Fatal("new_beacon_storing", err)
 		}
 		lastBeacon = newB
-		c.l.Info("NEW_BEACON_STORED", newB.String())
+		// measure time discrepancy in milliseconds
+		actual := time.Now().UnixNano()
+		expected := chain.TimeOfRound(c.conf.Group.Period, c.conf.Group.GenesisTime, newB.Round) * 1e9
+		discrepancy := float64(actual-expected) / float64(time.Millisecond)
+		c.l.Info("NEW_BEACON_STORED", newB.String(), "time_discrepancy_ms", discrepancy)
```

Comment:
can we make a Prometheus metric to track this as well?

Reply:
Done.